canvg.js

```diff
@@ -231,8 +231,14 @@
 			}
 			else if (window.DOMParser)
 			{
-				var parser = new DOMParser();
-				return parser.parseFromString(xml, 'text/xml');
+				try {
+					var parser = new DOMParser();

```

Comment:
you can keep `parser =` declared outside the try/catch and only do the `parseFromString` in it ... then i'll merge